### ISSUE TITLE
Problem with latest librsvg version

### ISSUE THREAD

**Issue by willforde** [picons/picons-source#89]
I am running Arch Linux and just updated the librsvg package to version 1:2.40.11-1.
After the update some TV Logos are unable to properly scale.
I was able to fix it again by downgrading back to version 1:2.40.9-2.

This might only be a problem with Arch but it might not.
Below is what BBC 2 should look like and what it actually looks like.

![bbctwo_good](https://cloud.githubusercontent.com/assets/558396/10838921/070394b8-7ec5-11e5-9813-124494abb187.png)

![bbctwo_bad](https://cloud.githubusercontent.com/assets/558396/10838923/08ab9612-7ec5-11e5-8745-11bbe8675eb1.png)

**Comment by charlesduco** [comment]
Can you provide an example of an svg logo that actually renders okay with librsvg 1:2.40.11-1?

**Comment by picons** [comment]
Shouldn't this be an issue for the developers of librsvg?

**Comment by charlesduco** [comment]
Yes, I investigated this and it's definitely a bug in one of the latest versions of librsvg
I made a simple, clean and valid svg file: https://gist.github.com/charlesduco/c0a27f148cd2e8adffa8
And rendered this with the following command in Ubuntu and Arch Linux:
`rsvg-convert -w 400 -h 400 -a -f png -o cat.png cat.svg`

#### The results:
##### Ubuntu - librsvg 2.40.2-1:
the image is scaled to 400x400
![cat_ubuntu](https://cloud.githubusercontent.com/assets/2472660/10854938/38d29f14-7f3f-11e5-95ae-229a24df1a9e.png)
##### Arch Linux - librsvg 1:2.40.11-1:
the viewport is scaled to 400x400, but the image stays the same width and height
![cat_arch](https://cloud.githubusercontent.com/assets/2472660/10854943/3d3cd97a-7f3f-11e5-9038-ccbb623185b7.png)
_________________________________________________________________________________
There is already a bug report at https://bugzilla.gnome.org/show_bug.cgi?id=756340
